devtools/cli/src/main/java/io/quarkus/cli/Build.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.cli;
+
+import java.util.concurrent.Callable;
+
+import io.quarkus.cli.core.ExecuteUtil;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "build", mixinStandardHelpOptions = false, description = "Build your quarkus project")
+public class Build implements Callable<Integer> {
+    @CommandLine.Option(names = { "-h", "--help" }, usageHelp = true, description = "display this help message")
+    boolean usageHelpRequested;
+
+    @CommandLine.ParentCommand
+    QuarkusCli parent;
```

Comment:
all this parent/spec code and possibly also the error handling we could move to a base class and simplify these commands quite a bit.